src/database/models/user.ts

```diff
@@ -191,6 +192,12 @@ User.init({
       unique: true,
       defaultValue: DataTypes.UUIDV4
   },
+  paidUntil: {
+      field: 'user_paid_until',
+	    type: DataTypes.DATE,
```

Comment:
NIT: spacing is off

Reply:
it seems that spacing is off throughout this file - spaces: 2 and not tabs: 4?